### DISCUSSION TITLE
discord links are invalid

### DISCUSSION THREAD

**Discussion by bootlegcoal** [omnivore-app/omnivore#3257] [General]
Hi, I tried clicking on it from a few places but all were invalid. fyi.

**Comment by IamSmeagol** [comment]
Could you be more specific? 

The links on the readme, welcome email, and docs worked for me

**Comment by jacksonh** [comment]
At lot of people have reported the Discord links "not working" in the past, but then managed to join using the link. You might want to try copying into your address bar in your browser.
